src/CMakeLists.txt

```diff
@@ -611,10 +611,13 @@ IF(WIN32)
           SET(VIRTUAL_KBD_SETTING "--no-virtualkeyboard")
      ENDIF()
 
-     IF("${QT_PATHS}" STREQUAL "")
-          SET(_qt_paths "")
+     IF(STELLARIUM_BUILD_ARM64)
+          cmake_path(APPEND QT_PATHS ${QT6_INSTALL_PREFIX} "bin" "qtpaths.bat")
```

Comment:
Now that we have STELLARIUM_BUILD_ARM64 we can locally generate the QT_PATHs value without the need of explicitly set by the user

Reply:
And welp. Workaround neeeded.